pubspec.lock

```diff
@@ -7,21 +7,21 @@ packages:
       name: _fe_analyzer_shared
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "61.0.0"
+    version: "54.0.0"
   analyzer:
     dependency: "direct main"
     description:
       name: analyzer
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "5.13.0"
+    version: "5.6.0"
```

Comment:
It looks like dependabot dart is doing some strange things

analyzer `v5.13.0` requires dart `2.19.0`, and dependabot happily bumped to that version: https://github.com/Workiva/scip-dart/pull/75

But, as can be seen from the pubspec.yaml, the min dart version is still set to `">=2.18.0 <3.0.0"`, implying dependabot is ignoring this specification

This means that when dependabot directly changes just the pubspec.lock file, its bumping the lockfile, but anytime you re-run pub get from the codebase, pub will correctly adjust the version used